powerstore/resource_host_group.go

```diff
@@ -138,6 +138,13 @@ func (r *resourceHostGroup) Create(ctx context.Context, req resource.CreateReque
 		return
 	}
 
+	if plan.HostConnectivity.ValueString() != "" {
```

Comment:
Do we have to provide HostConnectivity with " for create operation mandatorily?

Reply:
Then we can simply check for !IsNull() for hostconnectivity?

Reply:
checked behavior with `if !plan.HostConnectivity.IsNull()` but this always passes and throws an error even if host_connectivity is not there in the resource block. behavior of `if !plan.HostConnectivity.IsUnknown()` is similar to `if plan.HostConnectivity.ValueString() != ""`